collections/_article/multiplayer-in-godot-4-0-scene-replication.md

```diff
@@ -0,0 +1,329 @@
+---
+title: "Multiplayer in Godot 4.0: Scene Replication"
+excerpt: "Create multiplayer games in an instance (pun intended) with the new MultiplayerSpawner and MultiplayerSynchronizer nodes.
+Check out the key concepts, and get started with a quick tutorial on how to make a simple game using Godot multiplayer features!"
+categories: ["progress-report"]
+author: Fabio Alessandrelli
+image: /storage/blog/covers/multiplayer-in-godot-4-0-scene-replication.png
+date: 2023-02-18 18:00:00
+---
+
+Howdy Godotters! It's been more then a year since I last wrote on this blog, "... blah blah, blah blah, blah" (cf. C. L. C. Chuckie), and with Godot 4.0 nearing release, it s time to write something to jumpstart you into creating your own multiplayer experience with Godot.
```

Comment:
```suggestion
Howdy Godotters! It's been more than a year since I last wrote on this blog, "... blah blah, blah blah, blah" (cf. C. L. C. Chuckie), and with Godot 4.0 nearing release, it s time to write something to jumpstart you into creating your own multiplayer experience with Godot.
```